File: python/e621.py
# -*- coding: utf-8 -*-
"""
Created on Fri Jan  3 00:26:51 2020

@author: Username
"""

import sys

for t in range(sys.stdin):
    a,b,c=list(map(int, input().replace('\n','').split(' ')))
    tes=False
    for i in range(a+1,b):
        if i%c:
            tes=True
            print(i,end=' ')
    if tes: print()
    else: print('No free parking spaces.')

File: c++/e621.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    short t; cin >> t;
    while (t--)
    {
        short a,b,c; cin >> a >> b >> c;
        bool tes=false;
        for (int i=a+1;i<b;i++){
            if (i%c) { tes=true;cout << i << " ";}
        }
        if (tes) cout << "\n";
        else cout << "No free parking spaces.\n";
    }
}
